src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3676,4 +3676,7 @@
   <data name="Argv_IncludeDoubleQuote" xml:space="preserve">
     <value>The argv[0] argument cannot include a double quote.</value>
   </data>
-</root>
+  <data name="ResourceManager_ReflectionNotAllowed" xml:space="preserve">
+    <value>Use of ResourceManager for non primitive types is disabled. Set the MSBuild Property CustomTypeResourcesSupport to true in order to enable it.</value>
```

Comment:
```suggestion
    <value>Use of ResourceManager for custom types is disabled. Set the MSBuild Property CustomTypeResourcesSupport to true in order to enable it.</value>
```
This is both for consistency and correctness. The resource reader handles more than just what is the common definiton of primitive types.